ckanext/geocat/utils/xpath_utils.py

```diff
@@ -113,21 +113,23 @@ def xpath_get_first_of_values_from_path_list(node, path_list, get=XPATH_NODE):
 
 def xpath_get_language_dict_from_geocat_multilanguage_node(node):
     language_dict = {'en': '', 'it': '', 'de': '', 'fr': ''}
-    try:
+    localised_string_found = False
+    for locale in LOCALES:
+        value_locale = \
+            node.xpath('.//gmd:textGroup/gmd:LocalisedCharacterString[@locale="#{}"]/text()'  # noqa
+                       .format(locale),
+                       namespaces=gmd_namespaces)
+        if value_locale:
+            localised_string_found = True
+            language_dict[locale.lower()] = _clean_string(value_locale[0])
+    if localised_string_found:
+        return language_dict
+    value = node.xpath('.//gco:CharacterString/text()',
+                       namespaces=gmd_namespaces)
+    if value:
         for locale in LOCALES:
-            value_locale = \
-                node.xpath('.//gmd:textGroup/gmd:LocalisedCharacterString[@locale="#{}"]'  # noqa
-                           .format(locale) + '/text()',
-                           namespaces=gmd_namespaces)
-            if value_locale:
-                language_dict[locale.lower()] = _clean_string(value_locale[0])
+            language_dict[locale.lower()] = _clean_string(value[0])
```

Comment:
Since `_clean_string(value[0])` is the same for all locales, could we move this outside of the for-loop, so we don't have to do the cleaning four times?